### PR TITLE
 Do not flat in to map object or array attributes of event

### PR DESCRIPTION
**Pull request by AlvaroVega**
issue https://github.com/telefonicaid/perseo-fe/issues/720
And reduce footprint of event

### CODE REVIEW

**Comment by fgalan** [comment]
Should documentation be modified due to this PR?

**Comment by AlvaroVega** [comment]
> Should documentation be modified due to this PR?

Doc updated in https://github.com/telefonicaid/perseo-fe/pull/721/commits/8aa53a16cffea6542bc224de471497b889a54b1c

File: docs/API/plain_rules.md [hunk 1]
```diff
@@ -917,117 +917,5 @@ could be used by a rule so
...
-    "subservice": "/",
-    "service": "unknownt",
-    "myJsonValue__color": "blue",
-    "myJsonValue": {"type":"myType1","value":{"color":"blue"}},
-    "myArrayValue__0": "green",
-    "myArrayValue__1": "black"
-    "myArrayValue": { "type": "myType2", "value": ["green", "blue"] },
-    "location__type": "Point",
-    "location":{"type":"Point","coordinates":[53.120405283,53.0859375]},
-    "stripped": {
```
Comment (fgalan) [inline docs/API/plain_rules.md:1007]: I understand `stripped` was removed time ago 

```
- Remove unnecessary event data transformations (stripped, location parse, datetime parse) (#706, #712)
```

but documentation hasn't been fixed until this PR.

NTC

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,3 +1,4 @@
+- Do not flat in to map object or array attributes of event (#720)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: do not flat in to map object or array attributes of event (#720)
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**